include/effectmanager.h

```diff
@@ -84,7 +80,7 @@ class  EffectManager : public IJSONSerializable
     bool _clearTempEffectWhenExpired = false;
     bool _newFrameAvailable = false;
 
-    std::shared_ptr<GFXTYPE> * _gfx;
+    std::vector<std::shared_ptr<GFXTYPE>> _gfx;

```

Comment:
Could also be psram type

Reply:
Do you mean the shared_ptrs to the GFX objects? Technically yes, but I'm not sure how much we would gain from that?